roles/openshift_common/meta/main.yml

```diff
@@ -12,7 +12,7 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
-- { role: os_firewall }
-- { role: openshift_facts }
-- { role: openshift_repos }
-- { role: openshift_cloud_provider }
+- role: os_firewall
+- role: openshift_facts
+- role: openshift_repos
+- role: openshift_cloud_provider
```

Comment:
Should openshift_cloud_provider be a dependency here, or on the master/node roles instead? The opeshift_cli role for example wouldn't need cloud_provider config...